crates/net/network/src/network.rs

```diff
@@ -68,7 +72,12 @@ impl NetworkHandle {
             chain_id,
             tx_gossip_disabled,
             discv4,
+            event_listeners: Default::default(),
         };
+        let _ = inner
+            .to_manager_tx
+            .send(NetworkHandleMessage::EventListener(inner.event_listeners.clone()));
```

Comment:
I think this is redundant because we have access the channel from the manager directly